MineLeapMod/build.gradle

```diff
@@ -52,6 +52,7 @@ processResources {
 
 }
 
+runClient.outputs.upToDateWhen {false}

```

Comment:
This is intended to keep the runClient tast out of date so even if you run it twice in a row, you will never get "UP-TO-DATE"